backend/apps/views/__init__.py

```diff
@@ -0,0 +1 @@
+from .organizations import *
```

Comment:
For Django do we have to do explicit imports within `__init__.py`? For packages I read a while ago that this should not be done.

Reply:
Well i think this will probably change with the use of Django REST framework

Reply:
Totally fine then :) Just checking based on prior experiences and the love of not having import statements in Nuxt ❤️🙃